types/api/xverse/coins.ts

```diff
@@ -15,3 +15,12 @@ export interface SignedUrlResponse {
 }
 
 export type CoinsResponse = Coin[];
+
+export type Brc20Token = {
```

Comment:
did you consider extending BaseToken like Coin does?
```suggestion
export interface Brc20Token extends BaseToken = {
```

Reply:
`ticker` is optional in `BaseToken` but non-optional in `Brc20Token`. Might have to redeclare it.

Reply:
yup done